libs/dh/wholesale/feature-calculation-steps/src/lib/steps/consumption-per-energy-supplier.component.scss

```diff
@@ -14,3 +14,19 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+.headline,
+.sub-headline {
+  margin: 0;
+}
+
+.sub-headline {
+  // TODO: We don't have any spacing variable for 24px!
+  margin-bottom: 24px;
+  padding-bottom: 24px;
+  border-bottom: 1px solid var(--watt-color-neutral-grey-300);
+}
+
+watt-spinner {
+  display: flex;
+  justify-content: center;
+}
```

Comment:
question: there is some duplication in these two steps, which could advocate for creating some shared components to help with this. but i figured this would be easier to do once we have a few more steps, as otherwise we are probably not going to get the abstraction right in the first place. do you agree? 🙂

Reply:
Yea, I agree, but this is something we should look into for the next step to avoid inconsistency across the drawers